be/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM node:16.13.0 as build-stage
```

Comment:
The idea of a multi-stage build is to have a reduced final image.

At some point you should copy the generated/compiled code into a new image.

Have a look at
https://github.com/nodejs/docker-node/blob/main/docs/BestPractices.md

Reply:
The generated code is still in another PR, but when it is on master I will adapt it to copy the gen code.